framework/src/reporter/multi-reporter.ts

```diff
@@ -5,8 +5,11 @@ interface ReporterList {
   addReporter: (reporter: Reporter) => void;
 }
 
+interface MultiReporterType extends ReporterList, Reporter {
```

Comment:
please extract that into a separate file, just for cleanliness

Reply:
Refactored as desired.